clang/include/clang/Basic/Attr.td

```diff
@@ -1239,6 +1239,23 @@ def SYCLSimdAccessorPtr : InheritableAttr {
   let Documentation = [Undocumented];
 }
 
+// The attribute denotes that it is a function written in SIMT fashion, which
```

Comment:
We should not say that the function is written in SIMT fashion, as SIMT fashion allows workitems to have independent forward progress, in which case we'd require developer to use subgroup barriers for synchronization in some cases. Instead, we assume this function executes each operator in lock-step after vectorization (widening). Please adjust the comment.

Reply:
I can replace the word "SIMT" with "scalar". Would that be sufficient? I don't think we need to dig too much into the details here.

Reply:
scalar is ok